transformations/color_transformation/colors.json

```diff
@@ -0,0 +1,737 @@
+        "name": "light blue"
+    },
+    "LightCoral": {
+        "decimal": "240,128,128",
+        "hex": "#F08080",
+        "name": "light coral"
+    },
+    "LightCyan": {
+        "decimal": "224,255,255",
+        "hex": "#E0FFFF",
+        "name": "light cyan"
+    },
+    "LightGoldenRodYellow": {
+        "decimal": "250,250,210",
+        "hex": "#FAFAD2",
+        "name": "light golden rod yellow"
+    },
+    "LightGray": {
+        "decimal": "211,211,211",
+        "hex": "#D3D3D3",
+        "name": "light gray"
+    },
+    "LightGreen": {
+        "decimal": "211,211,211",
+        "hex": "#D3D3D3",
+        "name": "light green"
+    },
+    "LightGrey": {
+        "decimal": "144,238,144",
+        "hex": "#90EE90",
+        "name": "light grey"
+    },
+    "LightPink": {
+        "decimal": "255,182,193",
+        "hex": "#FFB6C1",
+        "name": "light pink"
+    },
+    "LightSalmon": {
+        "decimal": "255,160,122",
+        "hex": "#FFA07A",
+        "name": "light salmon"
+    },
+    "LightSeaGreen": {
+        "decimal": "32,178,170",
+        "hex": "#20B2AA",
+        "name": "light sea green"
+    },
+    "LightSkyBlue": {
+        "decimal": "135,206,250",
+        "hex": "#87CEFA",
+        "name": "light sky blue"
+    },
+    "LightSlateGray": {
+        "decimal": "119,136,153",
+        "hex": "#778899",
+        "name": "light slate gray"
+    },
+    "LightSlateGrey": {
+        "decimal": "119,136,153",
+        "hex": "#778899",
+        "name": "light slate grey"
+    },
+    "LightSteelBlue": {
+        "decimal": "176,196,222",
+        "hex": "#B0C4DE",
+        "name": "light steel blue"
+    },
+    "LightYellow": {
+        "decimal": "255,255,224",
+        "hex": "#FFFFE0",
+        "name": "light yellow"
+    },
+    "Lime": {
+        "decimal": "0,255,0",
+        "hex": "#00FF00",
+        "name": "lime"
+    },
+    "LimeGreen": {
+        "decimal": "50,205,50",
+        "hex": "#32CD32",
+        "name": "lime green"
+    },
+    "Linen": {
+        "decimal": "250,240,230",
+        "hex": "#FAF0E6",
+        "name": "linen"
+    },
+    "Magenta": {
+        "decimal": "255,0,255",
+        "hex": "#FF00FF",
+        "name": "magenta"
+    },
+    "Maroon": {
+        "decimal": "128,0,0",
+        "hex": "#800000",
+        "name": "maroon"
+    },
+    "MediumAquaMarine": {
+        "decimal": "102,205,170",
+        "hex": "#66CDAA",
+        "name": "medium aqua marine"
+    },
+    "MediumBlue": {
+        "decimal": "0,0,205",
+        "hex": "#0000CD",
+        "name": "medium blue"
+    },
+    "MediumOrchid": {
+        "decimal": "186,85,211",
+        "hex": "#BA55D3",
+        "name": "medium orchid"
+    },
+    "MediumPurple": {
+        "decimal": "147,112,219",
+        "hex": "#9370DB",
+        "name": "medium purple"
+    },
+    "MediumSeaGreen": {
+        "decimal": "60,179,113",
+        "hex": "#3CB371",
+        "name": "medium sea green"
+    },
```

Comment:
Also, if the decimal and hex values are not being used, you might as well remove them?

Reply:
Sounds good! I will keep the dict structure since it's probably easier to add things later.